packages/manager/modules/vrack/src/index.js

```diff
@@ -18,26 +19,29 @@ import './vrack-mapper.less';
 
 import actionsPartials from './partials/actions.html';
 import availablePartials from './partials/available.html';
-import controller from './vrack.controller';
+import component from './vrack.component';
 import mappedPartials from './partials/mapped.html';
 import routing from './vrack.routing';
 import vrackAdd from './add';
+import vrackMoveDialog from './move-dialog';
 
 const moduleName = 'ovhManagerVrack';
 
 angular
   .module(moduleName, [
     'ui.router',
     'ui.bootstrap',
+    'oui',
     'ovh-api-services',
     'ovhManagerCore',
     'ngOvhCloudUniverseComponents',
     'ngOvhToaster',
     'ui.router',
     vrackAdd,
+    vrackMoveDialog,
   ])
+  .component('ovhManagerVrackComponent', component)
```

Comment:
This doesn't cause any issue with `ng-include` and `$scope` inheritance ?

Reply:
The included html still inherits the parent's scope properly and things are working :)
Did you expect some issue here?

Reply:
If the $scope inheritance is still working then there is no issue :) 
I was wondering since components has an isolated scope, you can have issue inheriting methods defined in parent scope but I guess here this component is the parent